MCprep_addon/spawner/effects.py

```diff
@@ -1038,10 +1047,7 @@ def effects_enum(self, context):
 			elist.append((
 				str(effect.index),
 				effect.name + short_type,
-				"Add {} {} from {}".format(
-					effect.name,
-					display_type,
-					os.path.basename(effect.filepath))
+				f"Add {effect.name} {display_type} from {os.path.basename(effect.filepath)}"
```

Comment:
To be honest, I think for this specific use case, I prefer having the old `.format` syntax. It allows us to put the variables on their own lines, which tends to be nicer in editors